examples/portworx/script_sc.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+response=$(curl -s -d '{"parameters": { "cloud_drive": "Yes"}, "region": "us-south"}' -H "Content-Type: application/json" -X POST https://px-osb-test-2.tapas-iks-1.us-east.containers.appdomain.cloud/sclist)
```

Comment:
add a TODO to update the URL for fetching storage classes here once cloud drive support is pushed to prod, also is the `region` parameter mandatory and if yes then how we determine the value?

Reply:
can we please add a sample response of running this script in the PR description if not already done